atomix/cluster/src/main/java/io/atomix/raft/storage/log/RaftLog.java

```diff
@@ -62,7 +61,7 @@ public static RaftLogBuilder builder() {
    * @return the reader
    */
   public RaftLogReader openReader() {
```

Comment:
:thought_balloon: Here I am surprised by the method names. I would expect a pair of `openComittedReader` and `openUncommittedReader` But instead the commit reader has a special name, but the uncommitted doesn't have one.

I find this confusing. To me, using the uncommitted reader seems more dangerous, so I would expect to be warned by the method name.